Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Office 2007/Bases/PaletteOffice2007Base.cs

```diff
@@ -619,7 +619,7 @@ public override Color GetBackColor1(PaletteBackStyle style, PaletteState state)
                     }
                     else
                     {
-                        return (state == PaletteState.Tracking) || (style == PaletteBackStyle.InputControlStandalone)
+                        return state == PaletteState.Tracking || style == PaletteBackStyle.InputControlStandalone
```

Comment:
What was wrong with the brackets ? At least is showed grouped logic

Reply:
Reverted ReSharper